### PR TITLE
Update yarn.lock

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

I ran `yarn install --dev` on master as I often do on each breanch when switching for PR reviews and I noticed this time it updated the lock file, which makes me suspect it wasn't updated correctly in a previous PR.

Try to do the same on your local master and see if the `yarn.lock` gets updated locally

### CODE REVIEW

**Comment by citizen428** [comment]
I noticed this yesterday too, but then forgot about it as I was deep in my (backend) PR and `config.webpacker.check_yarn_integrity = true` in `development.rb` was a good enough temp fix for me.

But I agree, we should probably update this file, as it seems it wasn't checked in last time packages were updated.

**Comment by nickytonline** [comment]
I had changes to yarn.lock the other day in regards to fixing Storybook (#5783), but made sure it updated correctly. Just ensure with these changes that you can still view Storybook (`npm run storybook`, then navigate to http://localhost:6006)

**Comment by rhymes** [comment]
@citizen428 I actually thought it was always `true` in development! @nickytonline, is there a reason to keep the yarn integrity check disabled in development?

@nickytonline:

> Just ensure with these changes that you can still view Storybook (npm run storybook, then navigate to http://localhost:6006

You mean `yarn storybook` ;-) ? Yeah, it works:

```shell
➜  devto git:(rhymes/update-yarn-lockfile) yarn storybook
yarn run v1.21.1
$ start-storybook -p 6006 -c app/javascript/.storybook -s app/javascript/.storybook/assets
info @storybook/react v3.4.11
info
info => Loading static files from: /Users/rhymes/Development/devto/app/javascript/.storybook/assets .
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (full-control mode).
webpack built 1f85e6ae6f8d02aec3e9 in 22575ms
info Storybook started on => http://localhost:6006/
info
```

Screenshot 2020-01-30 at 11 51 18 AM

**Comment by nickytonline** [comment]
> @nickytonline, is there a reason to keep the yarn integrity check disabled in development?

I have more experience with npm over yarn, but to me it makes sense to have it enabled all the time.

**Comment by rhymes** [comment]
I traced the optional check for integrity to this PR: 

https://github.com/thepracticaldev/dev.to/pull/296

see this discussion https://github.com/thepracticaldev/dev.to/pull/296#pullrequestreview-146374357

Basically we can't enable it by default, because it's broken under docker: https://github.com/rails/webpacker/issues/1568

I'm going to add a note in `development.rb` about it

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-  ENV.fetch("YARN_INTEGRITY_ENABLED", "true") == "true"
-end
-
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = yarn_integrity_enabled?
+  # As the integrity check is currently broken under Docker with webpacker,
+  # we can't enable this flag by default
+  # see <https://github.com/thepracticaldev/dev.to/pull/296#discussion_r210635685>
+  config.webpacker.check_yarn_integrity = ENV.fetch("YARN_INTEGRITY_ENABLED", "true") == "true"
```
Comment (citizen428) [inline config/environments/development.rb]: 👍 for inlining this.

**Pull request merged**